internal/config/yaml_parser.go

```diff
@@ -67,8 +74,11 @@ func ParseConfigFile(fileName string) (mountConfig *MountConfig, err error) {
 	}
 	// convert log severity to upper-case
 	mountConfig.LogConfig.Severity = LogSeverity(strings.ToUpper(string(mountConfig.LogConfig.Severity)))
-	if !IsValidLogSeverity(mountConfig.LogConfig.Severity) {
-		err = fmt.Errorf("error parsing config file: log severity should be one of [trace, debug, info, warning, error, off]")
+	if err = validateLogSeverity(mountConfig.LogConfig.Severity); err != nil {
+		return
```

Comment:
Don't we want to add additional error string about the specific error here, e.g.

```go
return fmt.Errorf("invalid log severity in config-file: %w", err)
```

The same goes for return from the validateWriteConfig error.